libs/blocks/chart/chart.js

```diff
@@ -487,22 +511,47 @@ export const getResponsiveSize = (authoredSize) => {
   return size;
 };
 
-/* c8 ignore next 19 */
+export const getLabelDegree = (chartStyles, isDesktop) => {
+  let diagonal = false;
+
+  if (isDesktop) {
+    diagonal = [...chartStyles]?.includes('desktop-diagonal-labels');
+  } else {
+    diagonal = [...chartStyles]?.includes('mobile-diagonal-labels');
+  }
+
+  return diagonal ? 60 : 0;
+};
+
+/* c8 ignore next 31 */
```

Comment:
Could you simplify this to:
```
export const getLabelDegree = (chartStyles, isDesktop) => {
  if ((isDesktop && [...chartStyles]?.includes('desktop-diagonal-labels'))
      || [...chartStyles]?.includes('mobile-diagonal-labels')) {
      return 60;
  }

  return 0;
};
```

Reply:
This doesn't mean precisely the same thing. After the OR operator it would need to check that it's not desktop and has the mobile class. I think how I have it is easier to read

Reply:
Good catch, could add `!isDesktop`. I am being nitpicky, but it seems odd to me to add a variable for a boolean when the return value of both of your conditional statements is the same and the checks you do on the array returns a boolean. 

Why not just do then:
```
export const getLabelDegree = (chartStyles, isDesktop) => {
  if (isDesktop) {
    if ([...chartStyles]?.includes('desktop-diagonal-labels')) return 60;
  } else {
    if([...chartStyles]?.includes('mobile-diagonal-labels')) return 60;
  }

  return 0;
};
```

Reply:
Good catch, could add !isDesktop. I am being nitpicky, but it seems odd to me to add a variable for a boolean when the return value of both of your conditional statements is the same and the checks you do on the array returns a boolean.

Why not just do then:

export const getLabelDegree = (chartStyles, isDesktop) => {
  if (isDesktop) {
    if ([...chartStyles]?.includes('desktop-diagonal-labels')) return 60;
  } else {
    if([...chartStyles]?.includes('mobile-diagonal-labels')) return 60;
  }

  return 0;
};

Reply:
I like having `diagonal` as a variable name to make it clear what the degree represents. Also, in my code the `60` degrees is listed only once which is preferable if we need to change it in the future. I don't see anything that breaks the styleguide or conventions so I'll keep it as is